src/conmon.c

```diff
@@ -84,6 +84,16 @@ int main(int argc, char *argv[])
 		exit(0);
 	}
 
+	char *notify_socket_path = getenv("NOTIFY_SOCKET");
+	if (notify_socket_path != NULL) {
+		setup_notify_socket(notify_socket_path);
+		int r = unsetenv("NOTIFY_SOCKET");
```

Comment:
how would `--sdnotify none` work if we always unset `NOTIFY_SOCKET`?

And who will set the bind mount inside the container when we won't pass `NOTIFY_SOCKET` to the OCI runtime?

Reply:
Podman does.

We unset this because we DON'T want the OCI runtime to have it.  Podman has already set a mount label and added a bind mount to the OCI spec. 

Essentially instead of setting NOTIFY_SOCKET in the env, podman sets a NOTIFY_SOCKET in the Environment part of the runtime spec, and adds a bind mount to the Mounts area.

That's part of this PR
https://github.com/goochjj/libpod/pull/1
(hence why you need it)

If you looked at it look again, I just rebased it on the most recent libpod-sd-notify-cmdline (which it's dependent on)

Reply:
If we turn this into a cmdline option, we just wouldn't set the NOTIFY_SOCKET in podman, and this wouldn't be necessary.

Reply:
Done